doc/man3/d2i_PrivateKey.pod

```diff
@@ -3,10 +3,14 @@
 =head1 NAME
 
 d2i_PrivateKey_ex, d2i_PrivateKey, d2i_PublicKey, d2i_KeyParams,
-d2i_AutoPrivateKey_ex,  d2i_AutoPrivateKey, i2d_PrivateKey, i2d_PublicKey,
-i2d_KeyParams, i2d_KeyParams_bio, d2i_PrivateKey_ex_bio, d2i_PrivateKey_bio,
-d2i_PrivateKey_ex_fp, d2i_PrivateKey_fp, d2i_KeyParams_bio, i2d_PrivateKey_bio,
-i2d_PrivateKey_fp
+d2i_AutoPrivateKey_ex,  d2i_AutoPrivateKey,
+i2d_PrivateKey_ex, i2d_PrivateKey,
+i2d_PublicKey_ex, i2d_PublicKey,
+i2d_KeyParams_ex, i2d_KeyParams, i2d_KeyParams_bio,
+d2i_KeyParams_bio,
+d2i_PrivateKey_ex_bio, d2i_PrivateKey_bio,
```

Comment:
Not sure the _ex in the middle of the name was a good idea (too late now).

Reply:
Yeah, that one caught me too, I was just too distracted with other stuff at the time...  but "too late"?  These are new in master, so "too late" isn't quite true, and that would be a fairly easy rename fest.  *Another PR*, though.